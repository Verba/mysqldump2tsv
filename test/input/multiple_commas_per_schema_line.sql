
DROP TABLE IF EXISTS `tests`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `tests` (
  `price` decimal(7,2) DEFAULT NULL,
  `name` varchar(255) NOT NULL
) ENGINE=InnoDB AUTO_INCREMENT=348 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;

--
-- Dumping data for table `tests`
--

LOCK TABLES `tests` WRITE;
/*!40000 ALTER TABLE `tests` DISABLE KEYS */;
INSERT INTO `tests` VALUES (12.50,'first test');
