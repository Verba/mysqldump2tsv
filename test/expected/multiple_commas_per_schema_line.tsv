price	name
12.50	first test
